### DISCUSSION TITLE
Add Silverstripe Platform push/pull

### DISCUSSION THREAD

**Discussion by Firesphere** [ddev/ddev#5801] [Ideas]
For users in the Silverstripe space, Silverstripe Platform is a quite common platform to deploy to, or pull data from.

Adding a "ssp" (short for Silverstripe Platform) pull/push in the same way as other platforms have (e.g. platform.sh), might be useful.

I would think it might be better in an addon, but I'm unsure about it.
Spec of the API is here: https://silverstripe.cloud/api

**Comment by rfay** [comment]
This will be very welcome, and can be done as an example, installed via an add-on, or integrated into DDEV. It's probably easier to maintain as an add-on, but it's easy to do anyway.
